packages/server/graphql/mutations/removePokerTemplateScaleValue.ts

```diff
@@ -54,7 +50,11 @@ const removePokerTemplateScaleValue = {
       .table('TemplateScale')
       .get(scaleId)
       .update((row) => ({
-        values: row('values').deleteAt(oldScaleValueIndex),
+        values: row('values').deleteAt(
+          row('values')
+            .offsetsOf(oldScaleValue)
```

Comment:
@tianrunhe i missed this in the code review, but here's a change that ensures the operation is atomic. kind of a strange pattern, but useful if you ever have to modify a value inside an array